lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -188,12 +188,14 @@ void CommandObjectDWIMPrint::DoExecute(StringRef command,
     ValueObjectSP valobj_sp;
     std::string fixed_expression;
 
+    result.SetUserInput(expr);
```

Comment:
This is at least an oddly named API, since it's not at all what the user input.  For instance, I might very well have run:

dwim-print -a 1 -l swift -- some expression

but this API is going to claim "some expression" is the user input.